src/main/java/org/tron/core/db/TransactionTrace.java

```diff
@@ -1,61 +1,202 @@
 package org.tron.core.db;
 
+import static org.tron.common.runtime.vm.program.InternalTransaction.TrxType.TRX_CONTRACT_CALL_TYPE;
+import static org.tron.common.runtime.vm.program.InternalTransaction.TrxType.TRX_CONTRACT_CREATION_TYPE;
+import static org.tron.common.runtime.vm.program.InternalTransaction.TrxType.TRX_PRECOMPILED_TYPE;
+
 import org.tron.common.runtime.Runtime;
+import org.tron.common.runtime.vm.program.InternalTransaction;
 import org.tron.common.utils.Sha256Hash;
+import org.tron.core.Constant;
+import org.tron.core.capsule.AccountCapsule;
+import org.tron.core.capsule.ContractCapsule;
 import org.tron.core.capsule.ReceiptCapsule;
 import org.tron.core.capsule.TransactionCapsule;
 import org.tron.core.exception.ContractExeException;
 import org.tron.core.exception.ContractValidateException;
+import org.tron.core.exception.ReceiptException;
+import org.tron.core.exception.TransactionTraceException;
+import org.tron.protos.Contract.CreateSmartContract;
+import org.tron.protos.Contract.TriggerSmartContract;
+import org.tron.protos.Protocol.SmartContract;
+import org.tron.protos.Protocol.Transaction;
+import org.tron.protos.Protocol.Transaction.Contract.ContractType;
 
 public class TransactionTrace {
+
   private TransactionCapsule trx;
 
   private ReceiptCapsule receipt;
 
+  private Manager dbManager;
+
+  private CpuProcessor cpuProcessor;
+
+  private StorageMarket storageMarket;
+
+  AccountCapsule owner;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Explicitly declare the visibility for "owner". ([squid:S2039](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2039))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1217&issues=AWT0NMQ5vlEfaeH0ZGhJ&open=AWT0NMQ5vlEfaeH0ZGhJ)

[ISSUE_KEY]: AWT0NMQ5vlEfaeH0ZGhJ